src/components/places/places.tsx

```diff
@@ -18,8 +19,14 @@ function Places({ currentOffers, setOfferId }: CitiesProps): JSX.Element {
   const activeCity = useAppSelector(selectActiveCity);
   const sortOffers = useSortOffers(activeSort, currentOffers);
 
-  const handlePlaceCardMouseOver = (id: string) => setOfferId(id);
-  const handlePlaceCardMouseLeave = () => setOfferId('');
+  const handlePlaceCardMouseOver = useCallback(
+    (id: string) => setOfferId(id),
+    [setOfferId]
+  );
+  const handlePlaceCardMouseLeave = useCallback(
+    () => setOfferId(' '),
```

Comment:
```suggestion
    () => setOfferId(''),
```

Возможно нужная пустая строка?